com.hazelcast.jet.core.JobRestartWithSnapshotTest.stressTest

https://hazelcast-l337.ci.cloudbees.com/job/jet-pr-builder/com.hazelcast.jet$hazelcast-jet-core/2771/testReport/junit/com.hazelcast.jet.core/JobRestartWithSnapshotTest/stressTest/

```
Error Message

java.lang.AssertionError: no snapshot found

Stacktrace

java.util.concurrent.ExecutionException: java.lang.AssertionError: no snapshot found
	at com.hazelcast.jet.core.JobRestartWithSnapshotTest.stressTest(JobRestartWithSnapshotTest.java:428)
Caused by: java.lang.AssertionError: no snapshot found
	at com.hazelcast.jet.core.JobRestartWithSnapshotTest.waitForNextSnapshot(JobRestartWithSnapshotTest.java:360)
	at com.hazelcast.jet.core.JobRestartWithSnapshotTest.lambda$stressTest$12(JobRestartWithSnapshotTest.java:424)

```

--- a/hazelcast-jet-core/src/test/java/com/hazelcast/jet/core/JobRestartWithSnapshotTest.java
+++ b/hazelcast-jet-core/src/test/java/com/hazelcast/jet/core/JobRestartWithSnapshotTest.java
@@ -421,6 +421,7 @@ public class JobRestartWithSnapshotTest extends JetTestSupport {
         spawn(() -> {
             for (int i = 0; i < 10; i++) {
                 job.restart();
+                waitForFirstSnapshot(snapshotsMap, 3);
                 waitForNextSnapshot(snapshotsMap, 5);
                 Thread.sleep(500);
             }

